src/CoinbaseSmartWalletFactory.sol

```diff
@@ -41,17 +58,33 @@ contract CoinbaseSmartWalletFactory {
         }
     }
 
-    /// @dev Returns the deterministic address of the account created via `createAccount`.
+    /// @notice Returns the deterministic address of the account created via `createAccount()`.
+    ///
+    /// @param owners The initial set of owners that was / will be provided to `createAccount()`.
```

Comment:
```suggestion
    /// @param owners The initial set of owners provided to `createAccount()`.
```

Reply:
heh sorry @stevieraykatz so which is this an argument for, my edit?

Reply:
Yes! I think this qualifies as present tense despite the past-tense "provided"